app/sidekiq/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -104,9 +104,8 @@ def perform(submission_id) # rubocop:disable Metrics/MethodLength
                          Account.find_by(idme_uuid: submission.user_uuid)
 
           begin
-            submit_to_claims_api = submission.claims_api?
             # send submission data to either EVSS or Lighthouse (LH)
-            response = if submit_to_claims_api # not needed once fully migrated to LH
+            response = if submit_to_claims_api? # not needed once fully migrated to LH
```

Comment:
The variable, `submit_to_claims_api`, isn't defined. You'll need `submission.claims_api?`